index.js

```diff
@@ -208,11 +213,13 @@ function normalizeArguments(url, opts) {
 
 	opts = Object.assign(
 		{
-			protocol: 'http:',
 			path: '',
 			retries: 5
 		},
 		url,
+		{
+			protocol: url.protocol || 'http:' // Override both null/undefined with default protocol
+		},
 		opts
 	);
```

Comment:
`urlParseLax` returns `null` for all properties that weren't parsed, so `protocol: null` used to take over the default `protocol: 'http:'` all the time for protocol-less urls; this change remedies that.